pkg/execution/svccatadmission/internaldns.go

```diff
@@ -78,6 +78,20 @@ func InternalDNSAdmitFunc(ctx context.Context, microsServerClient microsServerCl
 		return nil, errors.Errorf("error fetching service central data for serviceName %q", serviceName)
 	}
 
+	if serviceCentralData == nil {
+		reason := fmt.Sprintf(
+			"namespace service %q does not exist in Service Central - should be impossible",
```

Comment:
"should be impossible" is misleading. As I commented above, someone deleted a service from Service Central, which is totally possible, but breaks our system.